packages/devtools_app/lib/src/charts/chart.dart

```diff
@@ -20,35 +35,43 @@ class Chart extends StatefulWidget {
   final ChartController controller;
 
   @override
-  ChartState createState() => ChartState();
+  ChartState createState() => ChartState(controller);
 }
 
 class ChartState extends State<Chart> with AutoDisposeMixin {
-  ChartController controller;
+  ChartState(this.controller);
```

Comment:
State objects should have zero argument constructors. Instead of doing this you should write

`ChartController get controller => widget.controller;`

Reply:
I had this originally, I thought you didn't like this?  I'll put it back to what I had done.